velox/common/encode/Base64.h

```diff
@@ -101,8 +102,19 @@ class Base64 {
   static size_t
   decode(const char* src, size_t src_len, char* dst, size_t dst_len);
 
- private:
+  static void
+  decode_url(const char* src, size_t src_len, char* dst, size_t dst_len);
```

Comment:
where is this used?

Reply:
Seems like I missed it while cherry-pick. I have cleaned it up now, squashed and pushed the changes. Ran the fuzzer test again, output is as follow:

```
I20230327 10:00:03.238423 8996625 ExpressionFuzzer.cpp:974] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230327 10:00:03.238437 8996625 ExpressionFuzzer.cpp:978] from_base64url 9859 100.00% 342482
I20230327 10:00:03.238456 8996625 ExpressionFuzzer.cpp:984] ==============================> Bottom 1 by number of rows processed
I20230327 10:00:03.238471 8996625 ExpressionFuzzer.cpp:986] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230327 10:00:03.238484 8996625 ExpressionFuzzer.cpp:991] from_base64url 9859 100.00% 342482
I20230327 10:00:03.238499 8996625 ExpressionFuzzer.cpp:1004] ==============================> All stats sorted by number of times the function was chosen
I20230327 10:00:03.238513 8996625 ExpressionFuzzer.cpp:1006] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230327 10:00:03.238528 8996625 ExpressionFuzzer.cpp:1010] from_base64url 9859 100.00% 342482
```